### PR TITLE
No attacks on ally

### PR DESCRIPTION
**Pull request by Sashank999**
#138 added.

### CODE REVIEW

File: torntools/views/settings/settings.html [hunk 1]
```diff
@@ -340,6 +340,8 @@ <h2>Preferences</h2>
 									</div>
 								</div>
 							</div>
+							<div class="option" id="profile-block_ally_attacks"><input type="checkbox" />Block attacks on faction mates and on ally faction members</div>
```
Comment (Mephiles) [inline torntools/views/settings/settings.html:343]: We usually add the "New!" text here but I'll go over them anyway before putting out a new version so it's not that big of a thing.

**Comment by Mephiles** [comment]
In case of such features could you provide screenshots after figuring out a solution (or even before if possible)? @Sashank999

File: torntools/views/settings/settings.html (hunk 1 above)
Comment (Sashank999) [reply torntools/views/settings/settings.html:343]: Added "New!".

**Comment by Sashank999** [comment]
It is actually the same that you could observe on Chedburn's account.
Chedburn's profile _ TORN - Google Chrome 10-Jan-21 1_46_39 PM

**Comment by Mephiles** [comment]
:+1:

**Pull request merged**